Managers/FrameMan.cpp

```diff
@@ -371,6 +382,24 @@ namespace RTE {
 		return 0;
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void FrameMan::CreatePresetColorTables() {
+		// Create RGB lookup table that supposedly speeds up calculation of other color tables.
+		create_rgb_table(&m_RGBTable, m_DefaultPalette, nullptr);
+		rgb_map = &m_RGBTable;
+
+		// Create transparency color tables. Tables for other blend modes will be created on demand.
+		for (int index = 0; index < TransparencyPreset::TransPresetCount; ++index) {
+			int presetBlendAmount = index * 5;
```

Comment:
Bad magic number. Either use a switch, a lookup table, or define a c_transparencyStepSize next to the enum.